src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const resultObject = {
+    result: 0,
+  };
+
+  resultObject.add = function (a, b) {
+    return a + b;
+  };
+
+  resultObject.subtract = function (a, b) {
+    return a - b;
+  };
+
+  resultObject.multiply = function (a, b) {
+    return a * b;
```

Comment:
The `multiply` method should use the `*=` operator to multiply the current result by the operand.